feature/manager/src/main/java/com/kinandcarta/create/proxytoggle/manager/view/composable/ProxyManagerScreen.kt

```diff
@@ -0,0 +1,315 @@
+        ) {
+            Column {
+                Spacer(Modifier.height(dimensionResource(R.dimen.default_margin)))
+                addressTextField()
+                Spacer(Modifier.height(dimensionResource(R.dimen.default_margin)))
+                portTextField()
+            }
+            Spacer(Modifier.width(dimensionResource(R.dimen.large_margin)))
+            buttonAndLabel()
+        }
+    }
+}
+
+@Composable
+fun ButtonAndLabel(
+    proxyEnabled: Boolean,
+    onToggleProxy: () -> Unit
+) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier
+    ) {
+        ProxyToggleButton(
+            proxyEnabled = proxyEnabled,
+            onClick = onToggleProxy
+        )
+        Spacer(Modifier.height(dimensionResource(R.dimen.default_margin)))
+        Text(
+            text = stringResource(
+                if (proxyEnabled) R.string.connected else R.string.disconnected
+            ).uppercase(),
+            color = if (proxyEnabled) MaterialTheme.colors.primary else BlueyGrey,
+            modifier = Modifier.clearAndSetSemantics {},
+            style = StatusLabelTextStyle
+        )
+    }
+}
+
+@Preview(name = "Connected (Light)", group = "ProxyManagerScreen")
+@Composable
+fun ProxyManagerScreenConnectedPreview() {
+    ProxyManagerScreenForPreview(proxyEnabled = true)
+}
+
+@Preview(name = "Connected (Dark)", group = "ProxyManagerScreen")
+@Composable
+fun ProxyManagerScreenConnectedPreviewDark() {
+    ProxyManagerScreenForPreview(darkTheme = true, proxyEnabled = true)
+}
+
+@Preview(name = "Unconnected (Light)", group = "ProxyManagerScreen")
+@Composable
+fun ProxyManagerScreenUnconnectedPreview() {
+    ProxyManagerScreenForPreview()
+}
+
+@Preview(name = "Unconnected (Dark)", group = "ProxyManagerScreen")
+@Composable
+fun ProxyManagerScreenUnconnectedPreviewDark() {
+    ProxyManagerScreenForPreview(darkTheme = true)
+}
+
+private const val PREVIEW_LANDSCAPE_WIDTH = 800
+private const val PREVIEW_LANDSCAPE_HEIGHT = 500
+
+@Preview(
+    name = "Connected (Light Landscape)", group = "ProxyManagerScreen Landscape",
+    device = Devices.AUTOMOTIVE_1024p,
+    widthDp = PREVIEW_LANDSCAPE_WIDTH,
+    heightDp = PREVIEW_LANDSCAPE_HEIGHT
+)
+@Composable
+fun ProxyManagerScreenConnectedPreviewLandscape() {
+    ProxyManagerScreenForPreview(useVerticalLayout = false, proxyEnabled = true)
+}
+
+@Preview(
+    name = "Connected (Dark Landscape)", group = "ProxyManagerScreen Landscape",
+    device = Devices.AUTOMOTIVE_1024p,
+    widthDp = PREVIEW_LANDSCAPE_WIDTH,
+    heightDp = PREVIEW_LANDSCAPE_HEIGHT
+)
+@Composable
+fun ProxyManagerScreenConnectedPreviewDarkLandscape() {
+    ProxyManagerScreenForPreview(
+        useVerticalLayout = false,
+        darkTheme = true,
+        proxyEnabled = true
+    )
+}
+
+@Preview(
+    name = "Unconnected (Light Landscape)", group = "ProxyManagerScreen Landscape",
```

Comment:
Minor: Disconnected instead of Unconnected? 😅

Reply:
+1